.github/workflows/lintdiff-test.yaml

```diff
@@ -26,7 +26,7 @@ defaults:
 
 jobs:
   lintDiff:
-    name: Lint Diff
+    name: "[TEST-IGNORE] Swagger LintDiff - Test"
```

Comment:
I'd revert this.  This workflow should only run on engsys PRs, so there should be no confusion.